fc/parsing/actions.py

```diff
@@ -87,7 +87,7 @@
     E.Floor: sympy.floor,
     E.Max: sympy.Max,
     E.Min: sympy.Min,
-    E.Rem: sympy.mod,
+    E.Rem: sympy.core.mod,
```

Comment:
What version of sympy was this introduced in? We should add sympy to our explicit list of dependencies (rather than via cellmlmanip) and specify the minimum that we support.

Reply:
actually @MichaelClerx  @jonc125 shouldn't this be `Mod` rather than `mod`?
```
>>> type(sympy.core.mod)
<class 'module'>
>>> type(sympy.Mod)
<class 'sympy.core.function.FunctionClass'>
>>> sympy.core.mod(25,6)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'module' object is not callable
>>> sympy.Mod(25,6)
1
>>>
```

Reply:
I don't know